src/main/java/teammates/ui/webapi/output/CourseInfo.java

```diff
@@ -1,10 +1,9 @@
-package teammates.ui.webapi.action;
+package teammates.ui.webapi.output;
 
 import java.util.List;
 import java.util.stream.Collectors;
 
 import teammates.common.datatransfer.attributes.CourseAttributes;
-import teammates.ui.webapi.output.ApiOutput;
 
 /**
  * Data transfer objects for {@link CourseAttributes} between controller and HTTP.
```

Comment:
The `CourseResponse` (Should rename to `CourseData`) and `CoursesResponse`(Should rename to `CoursesData`) should be in two separate files.